src/sharepoint/task/sharepoint-task.controller.js

```diff
@@ -10,45 +10,21 @@ angular
         }
 
         $onInit () {
-            this.$scope.taskDetails = [];
-            this.hasResult = false;
-            this.loaders = {
-                tasks: true,
-                pager: false
-            };
-
             this.getTasks();
         }
 
         getTasks () {
-            this.loaders.tasks = true;
             this.tasksIds = null;
 
             return this.sharepointService.getTasks(this.$stateParams.exchangeId)
-                .then((ids) => {
-                    this.tasksIds = ids;
-                    if (_.isArray(ids) && !_.isEmpty(ids)) {
-                        this.hasResult = true;
-                    }
-                })
+                .then((ids) => { this.tasksIds = ids.map((id) => ({ id })); })
```

Comment:
Lodash here again please :smiley:

Reply:
Map is actually es5 and supported by IE9, so there's no reason to avoid it.